www/md_base/src/app/common/directives/inspectdata/inspectdata.spec.coffee

```diff
@@ -0,0 +1,103 @@
+beforeEach module 'app'
+
+describe 'inspectdata', ->
+
+    $rootScope = $compile = null
+
+    injected = ($injector) ->
+        $compile = $injector.get('$compile')
+        $rootScope = $injector.get('$rootScope')
+
+    beforeEach inject injected
+
+    it 'should inspect data correctly', ->
```

Comment:
nornally you should test only one thing for each test.
you can factorize in beforeeach the initialization of the directive, and then test each feature in a different 'it'